src/media.js

```diff
@@ -665,7 +642,7 @@
             }, [audioReadyChannel.port2]);
 
             workerAudioRec.postMessage({ cmd: 'audioWorklet', msg: false });
-            workerAudioRec.onmessage = function (e) {
+            workerAudioRec.onmessage = (e) => {
```

Comment:
Codacy found an issue: ['workerAudioRec' is not defined.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=6361181)